photon-core/src/main/java/org/photonvision/vision/target/TargetModel.java

```diff
@@ -109,21 +109,35 @@ public enum TargetModel implements Releasable {
                     new Point3(Units.inchesToMeters(3.25), -Units.inchesToMeters(3.25), 0),
                     new Point3(-Units.inchesToMeters(3.25), -Units.inchesToMeters(3.25), 0)),
             Units.inchesToMeters(3.25 * 2)),
-    kAruco6in_16h5( // Nominal edge length of 200 mm includes the white border, but solvePNP corners
-            // do not
+    kAruco6in_16h5( // Corners used are only the internal tag black corners, not the full tag size
+            // (which would include the white boarder)
             List.of(
                     new Point3(Units.inchesToMeters(3), Units.inchesToMeters(3), 0),
                     new Point3(Units.inchesToMeters(3), -Units.inchesToMeters(3), 0),
                     new Point3(-Units.inchesToMeters(3), -Units.inchesToMeters(3), 0),
                     new Point3(Units.inchesToMeters(3), -Units.inchesToMeters(3), 0)),
             Units.inchesToMeters(3 * 2)),
-    k6in_16h5( // Nominal edge length of 200 mm includes the white border, but solvePNP corners
-            // do not
+    k6in_16h5(
             List.of(
                     new Point3(-Units.inchesToMeters(3), Units.inchesToMeters(3), 0),
                     new Point3(Units.inchesToMeters(3), Units.inchesToMeters(3), 0),
                     new Point3(Units.inchesToMeters(3), -Units.inchesToMeters(3), 0),
                     new Point3(-Units.inchesToMeters(3), -Units.inchesToMeters(3), 0)),
+            Units.inchesToMeters(3 * 2)),
+    // 2024 FRC tag. 6.5in inner tag, 8.125 overall
+    kAruco6p5in_36h11(
```

Comment:
can why some coordinates are negatives and others aren't negative be documented in some way (ex. a link that talks about the tag coordinates) or just a note saying why (really why we need a different target model between stock apriltag and aruco)

Reply:
Documentation was added in #894 (this PR hasn't been updated to master yet), and the Aruco model is removed in #916.

Reply:
cool cool